kbs/src/error.rs

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2023 by Alibaba.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//! This Error type helps to work with Actix-web
+
+use std::fmt::Write;
+
+use actix_web::{body::BoxBody, HttpResponse, ResponseError};
+use kbs_types::ErrorInformation;
+use log::error;
+use strum::AsRefStr;
+use thiserror::Error;
+
+const ERROR_TYPE_PREFIX: &str = "https://github.com/confidential-containers/kbs/errors";
```

Comment:
This isn't changed in this PR, so we can come back to it, but I think it's somewhat confusing to include this URL in the errors because it doesn't actually point to anything.

Reply:
Yes. We have an issue for this for a long time https://github.com/confidential-containers/trustee/issues/99

Reply:
yeah as you can see by my comment I hadn't realized how this was implemented. I guess it's pretty easy to adjust I'm just not sure where we should actually point the link or if we want one at all.

Reply:
Probably we could add a link to somewhere in confidentialcontainers.org